scripts/run_examples

```diff
@@ -35,15 +35,15 @@ while getopts "s:l:h" opt; do
 done
 
 # Run all examples.
-for language in ${languages}; do
-  examples="$(find examples -mindepth 2 -maxdepth 4 -type d -regex '.*/module.*/'"${language}"'$' | cut -d'/' -f2 | uniq)"
-  for example in ${examples}; do
-    if [[ "${example}" == "chat" ]]; then
-        "${SCRIPTS_DIR}/run_example" -s "${server}" -l "${language}" -e chat -- --test
-    elif [[ "${example}" == "trusted_information_retrieval" ]]; then
-        "${SCRIPTS_DIR}/run_example" -s "${server}" -l "${language}" -e trusted_information_retrieval -- --location 0.0,0.0
-    else
-        "${SCRIPTS_DIR}/run_example" -s "${server}" -l "${language}" -e "${example}"
-    fi
-  done
+examples="$(find examples -mindepth 2 -maxdepth 4 -type d -regex '.*/module.*/'"${application_language}"'$' | cut -d'/' -f2 | uniq)"
+for example in ${examples}; do
+  if [[ "${example}" == 'aggregator' ]]; then
+      "${SCRIPTS_DIR}/run_example" -s "${server}" -a "${application_language}" -e aggregator -- --bucket=test --data=1:10,2:20,3:30
+  elif [[ "${example}" == 'chat' ]]; then
+      "${SCRIPTS_DIR}/run_example" -s "${server}" -a "${application_language}" -e chat -- --test
+  elif [[ "${example}" == 'trusted_information_retrieval' ]]; then
+      "${SCRIPTS_DIR}/run_example" -s "${server}" -a "${application_language}" -c rust -e trusted_information_retrieval -- --latitude 0.0 --longitude 0.0
```

Comment:
Just looking over this, does this mean we no longer run the cpp client in our CI, and instead run the rust client? :)

Reply:
Yes, but this is only for TIR example